docs/product/relay/projects.mdx

```diff
@@ -108,3 +108,17 @@ An empty list rejects all origins. Use the default `["*"]` to allow all origins.
   }
 }
 ```
+
+`config.features`
+
+Enable product features that are not yet enabled by default.
```

Comment:
Can you also provide a list or link to features that are already enabled by default? As a user, it's confusing which features I need to manually config

Reply:
I rephrased this sentence to bring a bit more clarity. When product features ["graduate"](https://develop.sentry.dev/feature-flags/#after-launch-graduation) fully, we usually remove all traces of the feature flag, so there is no complete list of enabled features, unfortunately.